### DISCUSSION TITLE
Runelite crashes

### DISCUSSION THREAD

**Discussion by LiamWithGit** [runelite/runelite#18220] [Q&A]
Hi,

I keep getting runelite crashes, where runelite closes itself. I dont have this with any other applications. I'm on rly good pc specs, I've attempted running in safe mode, 32 bit client and 64 bit client updated my drivers/bios etc. Nothing changed still getting constistant crashes. Just now it crashed like 3 times in 20 minutes with me death each time so starting to get annoying. Was hoping someone has a solution. 

below my client log and the recent 3 crash logs i had: 

[client.log](https://github.com/user-attachments/files/16830414/client.log)
[jvm_crash_pid_21812_r.log](https://github.com/user-attachments/files/16830415/jvm_crash_pid_21812_r.log)
[jvm_crash_pid_2140_r.log](https://github.com/user-attachments/files/16830416/jvm_crash_pid_2140_r.log)
[jvm_crash_pid_19760_r.log](https://github.com/user-attachments/files/16830417/jvm_crash_pid_19760_r.log)

**Comment by cdfisher** [comment]
This is probably a duplicate of #10628, you can see some common troubleshooting steps near the bottom of that thread.

**Reply by LiamWithGit** [reply]
Hi cdfisher, I tried some of the solutions mentioned in #10628 and made several changes in bios like disabling intel speedstep but non of it worked for me i still crashed within like +-1 hour of playing. here's the most recent crash when i was trying intel speedstep disabled: 

[jvm_crash_pid_20432_r.log](https://github.com/user-attachments/files/16871087/jvm_crash_pid_20432_r.log)

U know anything else i can try?

**Reply by LiamWithGit** [reply]
@cdfisher Still dont have a fix i tried everything in #10268, here is a recent crash:
[jvm_crash_pid_10016_r.log](https://github.com/user-attachments/files/16993742/jvm_crash_pid_10016_r.log)

Does that tell u anything?
